services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -11,8 +11,6 @@
 import ch.qos.logback.classic.Logger;
 import ch.qos.logback.classic.spi.ILoggingEvent;
 import ch.qos.logback.core.read.ListAppender;
-import org.easymock.EasyMock;
-import org.easymock.IAnswer;
 import org.jetbrains.annotations.NotNull;
 import org.junit.Assert;
```

Comment:
Should the class be using ```org.junit.jupiter.api.Assertions``` (i.e. JUnit5) instead?

Reply:
done